documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -0,0 +1,390 @@
+# Migrating from v3 to v4
+
+Polaris v4 contains mostly internal changes and has simple migration path. This guide goes through the necessary steps to deal with breaking changes in your application.
+
+**Note:** This migration guide is for v3 but may apply for v2 as well.
+
+- [Testing](#testing)
+- [API Changes](#api-changes)
+  - [AppProvider](#appprovider)
+    - [i18n](#i18n)
+  - [Navigation](#navigation)
+    - [iconBody](#icon-body)
+  - [ChoiceList](#choicelist)
+    - [title](#title)
+  - [Card](#card)
+    - [secondaryFooterAction](#secondaryfooteraction)
+  - [Select](#select)
+    - [groups](#groups)
+  - [Icon](#icon)
+    - [source](source)
+    - [untrusted](#untrusted)
+- [Removed Exports/Components](#removed-exportscomponents)
+  - [WithContext](#withcontext)
+  - [WithRef](#withref)
+  - [Autocomplete.ComboBox.TextField](#autocompletecomboboxtextfield)
+  - [Autocomplete.ComboBox.OptionList](#autocompletecomboboxoptionlist)
+  - [Navigation.UserMenu](#navigationusermenu)
+  - [Modal.Dialog](#modaldialog)
+  - [Tabs.Panel](#tabspanel)
+- [Dependencies](#dependencies)
+- [Typescript](#typescript)
+  - [Config](#config)
+  - [Types](#types)
+
+## Testing
+
+We’ve migrated to [React’s new context API](https://reactjs.org/docs/context.html) while restructuring Polaris’ entire context structure. Using the Polaris test provider will allow you to keep up to date Polaris' internal contexts.
```

Comment:
```suggestion
We’ve migrated to [React’s new context API](https://reactjs.org/docs/context.html) while restructuring Polaris’ entire context structure. Using the Polaris test provider will allow you to keep up to date Polaris’ internal contexts.
```

Reply:
Is `React` the right descriptive word to use next to Polaris? Or is it `test provider`?